docs/pages/setup/reference/metrics.mdx

```diff
@@ -108,6 +108,7 @@ Now you can see the monitoring information by visiting several endpoints:
 | `reversetunnel_connected_proxies` | gauge | Teleport | Number of known proxies being sought. |
 | `rx` | counter | Teleport | Number of bytes received. |
 | `server_interactive_sessions_total` | gauge | Teleport | Number of active sessions. |
+| `teleport_build_info` | gauge | Teleport | Provides build information of Teleport. |
```

Comment:
I think we should expand the description since some users might see this as a counterintuitive use of a gauge. Maybe briefly describe the labels used and indicate that the value will always be `1`.

Reply:
Updated the help message and description. Let me know if there should be further changes.